### DISCUSSION TITLE
[Projects beta] beta Projects not showing in native iOS app

### DISCUSSION THREAD

**Discussion by innerpurpose** [community/community#9651] [Projects and Issues]
Love what you guys are doing with the new projects, but your iOS app only shows original projects and not the new project beta projects in the app. This would make the app useful 😊

**Comment by stuntbox** [comment]
This is one of the biggies that’s hamstringing me from using the app more. We’ve gone all-in on using the new project boards because they’re so much more useful. But without access to them on the native app it means interacting with them is a desktop-only experience right now (and therefore a lot slower to get stuff organized).

**Reply by innerpurpose** [reply]
I've switched out of the native app and back to using GH on mobile browser.. We would love to see new features available. Our devs are in Cuba and so many other companies have blocked access to their PM softwares, so we're all-in as well :) More stuff for them to iron out. I told my devops the other day that every time I see a big company fail in some way, it makes me feel that much better about our product and the care that is going into it.. We love GH and hope they continue to put resources into the PM side of their app!

**Comment by DanBradbury** [comment]
Have folks on the product that would like to engage with our board but need a mobile offering on iOS. We never had the classic boards so they are SOL for the time being? Am looking into the [Android app that might have the functionality for beta Projects](https://github.com/community/community/discussions/20065)
